### PR TITLE
docs: Added reference to VMobject.set_fill method

### PR DESCRIPTION
**Pull request by LedMeExplain**
## Changelog / Overview
I'm new at contribuiting so I made a small change by adding reference to VMobject.set_fill method.

## Motivation
I've been using Manim for almost a month and I've realized that some useful methods are not documented, so I'll be working on that.

## Explanation for Changes
Explaining all of the methods in VMobject class will make Manim more attractive to general public to use it.
[VMobject.set_fill](https://manimce--1749.org.readthedocs.build/en/1749/reference/manim.mobject.types.vectorized_mobject.VMobject.html#manim.mobject.types.vectorized_mobject.VMobject.set_fill)

## Checklist
- [x] I have read the [Contributing Guidelines](https://docs.manim.community/en/latest/contributing.html)
- [x] I have written a descriptive PR title (see top of PR template for examples)
- [x] I have written a changelog entry for the PR or deem it unnecessary
- [x] My new documentation builds, looks correctly formatted, and adds no additional build warnings

## Reviewer Checklist
- [ ] The PR title is descriptive enough
- [ ] The PR is labeled correctly
- [ ] The changelog entry is completed if necessary
- [ ] Newly added functions/classes either have a docstring or are private
- [ ] Newly added functions/classes have [tests](https://github.com/ManimCommunity/manim/wiki/Testing) added and (optional) examples in the docs
- [ ] Newly added documentation builds, looks correctly formatted, and adds no additional build warnings

### CODE REVIEW

File: manim/mobject/types/vectorized_mobject.py [hunk 1]
```diff
@@ -183,6 +185,25 @@ def update_rgbas_array(self, array_name, color=None, opacity=None):
...
     def set_fill(self, color=None, opacity=None, family=True):
+        """Set the fill color and fill opacity of a :class:`VMobject`.
+
+        Parameters
+        ----------
+        color :
+            Fill color of the :class:`VMobject`.
+        opacity : :class:`float`
+            Fill opacity of the :class:`VMobject`.
+
```
Comment (huguesdevimeux) [inline manim/mobject/types/vectorized_mobject.py:196]: I think you forgot to mention the `return` statement

**Review by huguesdevimeux — CHANGES_REQUESTED** [review]
SGTM, just a little thing missing 

Thanks for contributing !

**Comment by LedMeExplain** [comment]
Thanks for your time. I'm new at github so... How can I fix this? Do I need a to send a new pull request?

**Comment by LedMeExplain** [comment]
@huguesdevimeux 
it's working fine
[VMobject.set_fill #1749](https://manimce--1749.org.readthedocs.build/en/1749/reference/manim.mobject.types.vectorized_mobject.VMobject.html#manim.mobject.types.vectorized_mobject.VMobject.set_fill)

**Review by huguesdevimeux — APPROVED** [review]

**Comment by huguesdevimeux** [comment]
> Thanks for your time. I'm new at github so... How can I fix this? Do I need a to send a new pull request?

I see that you figured it out. SGTM, thanks for the contribution :D

File: manim/mobject/types/vectorized_mobject.py [hunk 2]
```diff
@@ -182,7 +184,44 @@ def update_rgbas_array(self, array_name, color=None, opacity=None):
             curr_rgbas[:, 3] = rgbas[:, 3]
         return self
 
-    def set_fill(self, color=None, opacity=None, family=True):
+    def set_fill(self, color=None, opacity=None, family=True) -> "VMobject":
+        """Set the fill color and fill opacity of a :class:`VMobject`
+        including its subMobjects.
```
Comment (behackl) [inline manim/mobject/types/vectorized_mobject.py:189]: The second part about the submobjects depends on the ``family`` parameter and should be mentioned separately.

```suggestion
        """Set the fill color and fill opacity of a :class:`VMobject`.
```

File: manim/mobject/types/vectorized_mobject.py [hunk 3]
```diff
@@ -182,7 +184,44 @@ def update_rgbas_array(self, array_name, color=None, opacity=None):
...
+
+        Parameters
+        ----------
+        color : optional
+            Fill color of the :class:`VMobject`.
+        opacity : :class:`float`, optional
+            Fill opacity of the :class:`VMobject`.
```
Comment (behackl) [inline manim/mobject/types/vectorized_mobject.py:196]: If you want to add type hints, please add them to the function header (`def set_fill(self, color=None, opacity: float = None, family: bool = True):`). Also, here is a suggestion for a short description for the third parameter:

```suggestion
        color
            Fill color of the :class:`VMobject`.
        opacity
            Fill opacity of the :class:`VMobject`.
        family
            If ``True``, the fill color of all submobjects is also set.
```

File: manim/mobject/types/vectorized_mobject.py [hunk 4]
```diff
@@ -182,7 +184,44 @@ def update_rgbas_array(self, array_name, color=None, opacity=None):
...
+                    square = Square().scale(2).set_fill(WHITE,1)
+                    circle1 = Circle().set_fill(GREEN,0.8)
+                    circle2 = Circle().set_fill(YELLOW) # No fill_opacity
+                    circle3 = Circle().set_fill(color = '#FF2135', opacity = 0.2)
+                    group = Group(circle1,circle2,circle3).arrange()
+                    self.add(square)
+                    self.add(group)
+
+        .. important::
+            All Mobjects are initialized with 0 fill_opacity. If you forget to set it, the fill color won't show.
```
Comment (behackl) [inline manim/mobject/types/vectorized_mobject.py:219]: There is currently movement regarding this behavior (see #1714), I'd suggest to remove this block here and deal with a clarification of the situation in #1714.

**Review by behackl — APPROVED** [review]
Looks good to me in general, left a few comments! Please address them, then this can be merged.

**Comment by LedMeExplain** [comment]
It's perfect 👍

**Review by behackl — APPROVED** [review]

**Pull request merged**